server/server/src/file_service.rs

```diff
@@ -88,6 +88,7 @@ pub async fn upsert_file_metadata(
 
     for update in request.updates {
         let new = update.new;
+
```

Comment:
```suggestion
```
Try to remove all standalone inconsequential changes like this, esp before final review.